decorators.go

```diff
@@ -621,6 +621,8 @@ type CommandWithExecute interface {
 // CommandWithExecuteDecorator is a decorator that sets the command execution.
 type CommandWithExecuteDecorator struct{}
 
+type CobraCtxCmd struct{}
```

Comment:
Since this is something internal to ecdysis, it might be good to unexport it.

Reply:
@hariso I'd need it to ensure this in the client (conduit CLI)

```golang
if cmd, ok := ctx.Value(ecdysis.CobraCtxCmd{}).(*cobra.Command); ok {
		return cmd.Help()
	}
	```

Reply:
Ah, right. It might be then good to have a "getter" and "setter" functions, similar to what we have for example in the Connector SDK with connector IDs, e.g. `ecdysis.GetCobraCmd(ctx)`.

Reply:
I love it, thank you. https://github.com/ConduitIO/ecdysis/pull/9/commits/6dcf59c78e81559f6bccfbd97eeb03157e881261